csharp/extractor/Semmle.Extraction.CSharp.Standalone/CsProjFile.cs

```diff
@@ -5,17 +5,30 @@
 
 namespace Semmle.BuildAnalyser
 {
+    interface IProjectFile
```

Comment:
It looks like this interface is only used with `CsProjFile`, so remove?

Reply:
Might as well remove it. I think ideally in future this would be useful to have different classes for different project types.